docs/source/tutorial/tutorial-detail-view.md

```diff
@@ -0,0 +1,242 @@
+
+In the Explorer tab, start by clicking the "New Tab" button in the middle operations section: 
+
+<img alt="the new tab button" class="screenshot" src="images/explorer_new_tab.png"/>
+
+A new tab will be added with nothing in it: 
+
+<img alt="The UI after adding a new tab" class="screenshot" src="images/explorer_added_empty_tab.png"/>
+
+In the left-hand column, click the word "Query" under "Documentation" to be brought to a list of possible queries: 
+
+<img alt="The list of possible queries" class="screenshot" src="images/explorer_query_list.png"/>
+
+Select the `launch` query by clicking the button next to it. Sandbox Explorer will automatically set up the query for you to use: 
+
+<img alt="What the launch query will look like immediately after adding it" class="screenshot" src="images/explorer_launch_query_start.png"/>
+
+First, change the name of the operation from "Query" to "Launch Details" - that will then reflect in the tab name and make it easier to tell which query you're working with:  
+
+<img alt="The renamed query" class="screenshot" src="images/explorer_details_renamed.png"/>
+
+Let's go through what's been added here: 
+
+- Again, we've added an operation, but this time it's got a parameter coming into it. This was added automatically by Sandbox Explorer because there is not a default value provided for the non-null `launchId` argument. 
+- The parameter is prefixed with a `$` for its name, and the type is indicated immediately after. Note that the `ID` type here has an exclamation point, meaning it can't be null. 
+- Within that operation, we make a call to the `launch` query. The `id` is the argument the query is expecting, and the `$launchId` is the name of the parameter we just passed in the line above. 
+- Again, there's blank space for you to add the fields you want to get details for on the returned object, which in this case is a `Launch`. 
+- Finally, at the bottom, the "Variables" section of the Operations panel has been expanded, and a dictionary has been added with a key of `"launchId"`. At runtime, this will be used to fill in the blank of the `$launchId` parameter. 
+
+
+> Note: GraphQL's assumptions about nullability are different from Swift's. In Swift, if you don't annotate a property's type with either a question mark or an exclamation point, that property is non-nullable.
+>
+> In GraphQL, if you don't annotate a field's type with an exclamation point, that field is considered *nullable*. This is because GraphQL fields are **nullable by default**.
+>
+> Keep this difference in mind when you switch between editing Swift and GraphQL files.
+
+Now, switch back to Sandbox Explorer. Start by using the checkboxes or typing to add the properties you're already requesting in the `LaunchList` query. One difference: Use `LARGE` for the mission patch size since the patch will be displayed in a much larger `ImageView`:
```

Comment:
```suggestion
Now, switch back to Sandbox Explorer. Start by using the checkboxes or typing to add the properties you're already requesting in the `LaunchDetails` query. One difference: Use `LARGE` for the mission patch size since the patch will be displayed in a much larger `ImageView`:
```
Looks like a copy-paste mistake.

Reply:
Nope, it's referring to adding the same properties that are in the LaunchList query to the LaunchDetails query (Fragments will be covered in a planned later chapter)